app/code/core/Mage/Catalog/Model/Product.php

```diff
@@ -217,7 +217,7 @@
  * @method $this setStoreIds(array $storeIds)
  * @method array getSwatchPrices()
  *
- * @method int getTaxClassId()
+ * @method string getTaxClassId()
```

Comment:
The  `backend_type`  of `tax_class_id` is `int`.  Is it more correct to remain as `int`?